MainWindow/Main.cs

```diff
@@ -92,7 +92,7 @@ public Main()
             #endregion
             #region Localize & Populate Fields
             string[] args = Environment.GetCommandLineArgs();
-            string filename = args.Length > 0 ? Path.GetFileNameWithoutExtension(args[0]).ToLower() : "";
+            string filename = args.Length > 0 ? Path.GetFileNameWithoutExtension(args[0])?.ToLower() : "";
```

Comment:
Not needed.  args[0] will always be the path of the executable, so it will never be null.

What's more, the ?. operator will return null, which would cause a build error below on filename.IndexOf.  Because this will never be the case, however, that's not a problem.